pandas/tests/plotting/test_datetimelike.py

```diff
@@ -1222,6 +1222,14 @@ def test_secondary_y_irregular_ts_xlim(self):
         self.assertEqual(left, ts_irregular.index.min().toordinal())
         self.assertEqual(right, ts_irregular.index.max().toordinal())
 
+    def test_plot_outofbounds_datetime(self):
```

Comment:
Does pandas have the equivolent of `@cleanup` from the mpl tests?  This is touching global state and can leak to other tests.